app/application/use_cases/events/create_event.py

```
@@ -169,6 +213,18 @@ async def create_event(
         entity = _event_result_to_entity(created)
         if trigger_workflows and self.workflow_engine:
             await self._trigger_workflows(entity, tenant_id)
+        if self._webhook_dispatcher:
+            asyncio.create_task(
+                self._webhook_dispatcher.dispatch(
+                    tenant_id, entity, subject.subject_type.value
+                )
+            )
```

Ruff (0.15.5)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)